Detectors/AOD/src/AODProducerWorkflowSpec.cxx

```diff
@@ -52,37 +53,50 @@ using GID = o2::dataformats::GlobalTrackID;
 namespace o2::aodproducer
 {
 
+// using global variables to pass parameters to extra table filler
+float tpcInnerParam = 0.f;
+uint32_t flags = 0;
+uint8_t itsClusterMap = 0;
+uint8_t tpcNClsFindable = 0;
+int8_t tpcNClsFindableMinusFound = 0;
+int8_t tpcNClsFindableMinusCrossedRows = 0;
+uint8_t tpcNClsShared = 0;
+uint8_t trdPattern = 0;
+float itsChi2NCl = -999.f;
+float tpcChi2NCl = -999.f;
+float trdChi2 = -999.f;
+float tofChi2 = -999.f;
+float tpcSignal = -999.f;
+float trdSignal = -999.f;
+float tofSignal = -999.f;
+float length = -999.f;
+float tofExpMom = -999.f;
+float trackEtaEMCAL = -999.f;
+float trackPhiEMCAL = -999.f;
```

Comment:
Instead of out-of-class global variables, better you use a an exchange structure (can be nested to this class),
e.g.
struct TrackInfo {
  float tpcInnerParam = 0.f;
  ...
};
which would created locally and passed to addToTracksExtraTable